ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -63,6 +68,23 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
       this.can.can('connect target', target),
     );
 
-    return { targets: targets, totalItems };
+    let targetsWithSessions = [];
```

Comment:
none blocking comment: a small comment of what this for is doing can be helpful, such as small comment in line 66. The name of the variable `targetsWithSessions` is already descriptive tho, that's good naming choice 😄

Reply:
Good idea! Comment added.